src/routes/LoginPage/LoginPage.js

```diff
@@ -30,6 +46,52 @@ export default class LoginPage extends Component {
       });
   };
 
+  handleUserCreateSubmit = async (e) => {
+    e.preventDefault();
+
+    if (this.verifyPassword(this.state.password, this.state.confirmPassword)) {
+      await UsersService.createUser({
+        firstName: this.state.firstName,
+        lastName: this.state.lastName,
+        email: this.state.email,
+        password: this.state.password,
+        address: this.getFullAddress(),
```

Comment:
I know the address is a string in our database, but I wonder if we plan to let the users edit their profile, including the address? It might be hard to handle if we need to convert this address string back to address, city and state. If it's not included in our project scope then you can ignore this.

Reply:
This is a good point and something I didn't consider. I think when I get to editing, I can parse a string with ', ' as the separator. If that doesn't work then, I can refactor this

Reply:
Personally I don't usually like to parse the string cause we can't control what users may put in it, and I would prefer updating the database table. But if it's too much trouble it's totally fine to do as you said.